build.sh

```diff
@@ -8,4 +8,8 @@ while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symli
 done
 ScriptRoot="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 
++# Run under sudo so we can set ulimit
++# See https://github.com/dotnet/core-eng/issues/14808
+sudo "$ScriptRoot/build-ulimit.sh"
```

Comment:
Does this actually work or are you getting lucky?  My understanding of sudo and execution scope is that the ulimit will go back tot he machine's default the minute sudo returns, and build-ulimit.sh is only setting it, not setting it then performing the actions that need it.

Reply:
Oh, it's only setting it for the scope of the execution? Then I must have gotten lucky and will need to move the rest of the code from build.sh into build-ulimit.sh after the ulimit update.

Reply:
I was thinking either that or just make the build-ulimit script a pass-through to the regular build.  I could be wrong though since I could've sworn when @dsplaisted was hitting this it was at or pretty close to 100%

Reply:
Looks right now, assuming it still passes